parkbenchcommon/broadcaster.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Joel Allen Luellwitz and Emily Frost
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Provides the broadcasting component of a filesystem-based IPC mechanism.
+
+We decided to use this system because we were unsure that Unix domain sockets were capable
+    of broadcasting atomically to multiple programs and D-Bus would have required a
+    substantial restructuring of the entire project.
```

Comment:
I'd say "...and using D-bus...". Also reverse indent should be two spaces according to JOEL-8.